go/oprf/groups/ecgroup/ec.go

```diff
@@ -205,7 +205,10 @@ type Point struct {
 
 // New returns a new point initialised to constants.Zero
 func (p Point) New(pog gg.PrimeOrderGroup) gg.GroupElement {
-	return Point{X: constants.Zero, Y: constants.Zero, pog: pog, compress: true}
+	return Point{
+		X:   new(big.Int).Set(constants.Zero),
+		Y:   new(big.Int).Set(constants.Zero),
+		pog: pog, compress: true}
```

Comment:
tiny nit: can the brace be moved to a new line, and then also move `compress` down a line? :)